.github/workflows/stale.yml

```diff
@@ -16,9 +16,9 @@ jobs:
         stale-issue-message: ':wave: Hi! This issue has been marked stale due to inactivity. If no further activity occurs, it will automatically be closed.'
         stale-pr-message: ':wave: Hi! This pull request has been marked stale due to inactivity. If no further activity occurs, it will automatically be closed.'
         stale-issue-label: 'stale'
-        exempt-issue-label: 'keep-open'
-        exempt-pr-label: 'keep-open'
         stale-pr-label: 'stale'
+        exempt-issue-labels: 'keep-open'
+        exempt-pr-labels: 'keep-open'
```

Comment:
![image](https://user-images.githubusercontent.com/33973272/95934917-9a9aef80-0d97-11eb-983d-3dc1c759d408.png)

Hey @jhonnattan123 Look `fix: Make exempt labels plural for stale action (#100)` was committed with this PR.  This changes is already merged in the master branch. Could you drop that commit from your branch. Thanks!